onshape-java/api-generator/src/main/java/com/onshape/api/generator/java/JavaLibraryTarget.java

```diff
@@ -187,10 +188,16 @@ public void finish(boolean commit) throws GeneratorException {
     }
 
     void build(File location) throws GeneratorException {
+        // Get credentials for tests
```

Comment:
Am I reading this correctly - that you can now supply the key parameters on the command line? So the api library builder can now invoke the tests with the parameters it has read from the .onshape-build file? And the tests will no longer also require the environment variables?

Reply:
Yes, I realised that since the tests are run in a Process we can set the env vars using the .onshape-build parameters.

Reply:
Excellent! That will help in the build process here because I won't have to continue to expose the keys through an environment variable to run the tests.